posthog/clickhouse/migrations/0143_log_entries_exchange.py

```diff
@@ -0,0 +1,11 @@
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.clickhouse.log_entries import LOG_ENTRIES_DISTRIBUTED_TABLE, LOG_ENTRIES_TABLE
+
+EXCHANGE_LOG_ENTRIES_SQL = f"""
+EXCHANGE TABLES {LOG_ENTRIES_TABLE} AND {LOG_ENTRIES_DISTRIBUTED_TABLE}
+"""
+
+operations = [
+    run_sql_with_exceptions(EXCHANGE_LOG_ENTRIES_SQL, node_roles=[NodeRole.DATA, NodeRole.COORDINATOR]),
```

Comment:
is distribute table atomic? 
https://clickhouse.com/docs/sql-reference/statements/exchange?

Reply:
PS if there are any mv using LOG_ENTRIES_DISTRIBUTED_TABLE, after the exchange they will push data into LOG_ENTRIES_TABLE.

Reply:
xD i was hit hard once by that and had to hand fix it ;)